presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastoreStats.java

```diff
@@ -43,6 +43,7 @@
     private final HiveMetastoreApiStats listPrivileges = new HiveMetastoreApiStats();
     private final HiveMetastoreApiStats grantTablePrivileges = new HiveMetastoreApiStats();
     private final HiveMetastoreApiStats revokeTablePrivileges = new HiveMetastoreApiStats();
+    private final HiveMetastoreApiStats getFields = new HiveMetastoreApiStats();
```

Comment:
Nit: move this up to be after `getTable` so that the field ordering is consistent with the method ordering